lib/restforce/middleware/raise_error.rb

```diff
@@ -26,7 +26,10 @@ def on_complete(env)
     end
 
     def message
-      "#{body['errorCode']}: #{body['message']}"
+      message = "#{body['errorCode']}: #{body['message']}"
+      message << "\nRESPONSE: #{JSON.dump(@env[:body])}"
```

Comment:
What would you think about attaching the raw body to the exception as an instance variable with accessor, rather than adding it to the message?

Reply:
👍 Yep! I could do that.